src/rx.cpp

```diff
@@ -661,15 +682,14 @@ void Aggregator::process_packet(const uint8_t *buf, size_t size, uint8_t wlan_id
 
             init_fec(new_session_data->k, new_session_data->n);
 
-            fprintf(stdout, "%" PRIu64 "\tSESSION\t%" PRIu64 ":%u:%d:%d\n", get_time_ms(), epoch, WFB_FEC_VDM_RS, fec_k, fec_n);
-            fflush(stdout);
 
+            WFB_LOG(LogLevel::ALWAYS, "%" PRIu64 "\tSESSION\t%" PRIu64 ":%u:%d:%d", get_time_ms(), epoch, WFB_FEC_VDM_RS, fec_k, fec_n);
```

Comment:
@vertexodessa Need to change level to INFO to be consistent with other machine-readable telemetry logs

Reply:
INFO will add prefix to the log, ALWAYS does not add the prefix
considering this, should I change the level?

Reply:
This is telemetry counters for python part of wfb-ng. If will be processes by `fprintf(stdout, ...` a few lines above. If you need then on the android side then set log level which will skip it on normal wfb-ng build